ADOPTERS.md

```diff
@@ -2,6 +2,7 @@
 
 This is the list of organisations that are using Cortex in **production environments** to power their metrics and monitoring systems. Please send PRs to add or remove organisations.
 
+* [Amazon Web Services (AWS)](https://aws.amazon.com/)
```

Comment:
Would it make more sense to link to https://aws.amazon.com/prometheus/ instead?

Reply:
done!